lib/Runtime/Library/TypedArray.cpp

```diff
@@ -1730,6 +1730,24 @@ namespace Js
         return JavascriptArray::CopyWithinHelper(nullptr, typedArrayBase, typedArrayBase, length, args, scriptContext);
     }
 
+    Var TypedArrayBase::GetKeysEntriesValuesHelper(Arguments& args, ScriptContext *scriptContext, LPCWSTR apiName, JavascriptArrayIteratorKind kind)
```

Comment:
>GetKeysEntriesValuesHelper [](start = 24, length = 26)

TypeArrayBase::ValidateTypedArray() exists for the same "ValidateTypedArray()" operation.
Do you want to use it or change it (and use it)?